nilearn/plotting/displays.py

```diff
@@ -946,6 +1024,138 @@ def draw_cross(self, cut_coords=None, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def add_graph(self, adjacency_matrix, node_coords,
+                  node_color=None, node_size=None,
+                  edge_cmap=None, edge_threshold=None,
+                  edge_kwargs=None, node_kwargs=None):
+        """Plot undirected graph on each of the axes
+
+            Parameters
+            ----------
+            adjacency_matrix: numpy array of shape (n, n)
+                represents the edges strengths of the graph. Assumed to be
+                a symmetric matrix.
+            node_coords: numpy array_like of shape (n, 3)
+                3d coordinates of the graph nodes in world space.
+            node_color: color or sequence of colors
+                color(s) of the nodes.
+            node_size: scalar or array_like
+                size(s) of the nodes in points^2.
+            edge_cmap: colormap
+                colormap used for representing the strength of the edges.
+            edge_threshold: str or number
+                If it is a number only the edges with a value greater than
+                edge_threshold will be shown.
+                If it is a string it must finish with a percent sign,
+                e.g. "25.3%", and only the edges with a abs(value) above
```

Comment:
I am now wondering what was the percentage relative to when specifying edge_threshold as a percentage. At the moment it is a percenage of the number of possible node edges i.e. n * (n - 1) / 2 if the adjacency matrix is of shape (n, n). I am leaning towards changing that to be a percentage of non zero edges. To spell it out even more, "25%" would mean I want to keep 25% of the non-zero edges with the highest absolute value.

Any further thoughts or objections?